modules/op-release-notes-1-14.adoc

```diff
@@ -326,3 +326,27 @@ With this update, {pipelines-title} General Availability (GA) 1.14.3 is availabl
 
 * Before this update, when you used the `tkn pr logs -f` command to view the logs for a running pipeline, the command line utility stopped responding, even if the pipeline run completed successfully. With  this update, the `tkn pr logs -f` command properly displays the log information and exits.
+
+[id="release-notes-1-14-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.4
+
+With this update, {pipelines-title} General Availability (GA) 1.14.4 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-4_{context}"]
+=== Fixed issues
+
+* Before this update, a large number of error messages referencing `tekton-pipelines-webhook.ConversionWebhook` could be logged. With this update, unneeded conversion webhook configuration for the `ClusterTask` and `StepAction` Custom Resource Definitions (CRDs) was removed, and such error messages are no longer logged.
+
+* Before this update, some configuration settings using the `options` sections in the `TektonConfig` custom resource (CR) did not work, because a race condition would occurr if the same setting was configured in both the `options` section and another field in the `TektonConfig` CR. With this update, the settings work.
+
+* Before this update, the {pipelines-shortname} console plugin pod did not move to the node specified using the `nodeSelector`, `tolerations`, and `priorityClassName` settings. With this update, the {pipelines-shortname} plugin pod moves to the correct node.
+
+* Before this update, some error messages were logged in the operator controller logs without the proper context information. With this update, error messages contain the required information.
+
+* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} operator controller would fail to create default resources in new namespaces correctly. With this update, the controller functions correctly.
+
+* Before this update, if you configured the Horizontal Pod Autoscaler (HPA) using the `options` section in the `TektonConfig` CR, any existing HPA was updated correctly but a new HPA was not created when required. With this update, Horizontal Pod Autoscaler configuration using the `options` section works correctly.
+
+* Before this update, if a user or an {pipelines-shortname} controller used the {pipelines-shortname} API to modify a pipeline run that was in the process of being started by {pac},  {pac} could stop and the log contained "panic" messages. With this update, the pipeline being started by {pac} can be modified concurrently.
+
+* Before this update, in {pac}, a concurrency limit setting of 0 was not interpreted as disabling the concurrency limit. With this update, a concurrency limit setting of 0 disables the concurrency limit.
```

Comment:
Up to you, but if the value of 0 here is specifically referring to a value you'd set in a file, I'd put the number in backticks:
```suggestion
* Before this update, in {pac}, a concurrency limit setting of `0` was not interpreted as disabling the concurrency limit. With this update, a concurrency limit setting of `0` disables the concurrency limit.
```